### ISSUE TITLE
Can't make it work with a flatpak Firefox from a flathub repo

### ISSUE THREAD

**Issue by resiehnnes** [harshadgavali/searchprovider-for-browser-tabs#61]
I guess Firefox does not see installed 'tabsearchproviderconnector', and most likely it can be fixed adding some exclusions in Flatseal, but I can't figure out how. Would be great if someone will help me out. I absolutely love this extension.

**Comment by harshadgavali** [comment]
Yes, sandboxed browsers can't see programs installed on host (https://github.com/flatpak/xdg-desktop-portal/issues/655).

There's workarround which creates hole in sandbox by allowing browser to run any executable on host system.
But cause it's very insecure, I won't list it here, sorry.
